src/sql/parts/notebook/outputs/factories.ts

```diff
@@ -80,6 +80,16 @@ export const javaScriptRendererFactory: IRenderMime.IRendererFactory = {
 	createRenderer: options => new widgets.RenderedJavaScript(options)
 };
 
+export const dataResourceRendererFactory: IRenderMime.IRendererFactory = {
+	safe: true,
+	mimeTypes: [
+		'application/vnd.dataresource+json',
+		'application/vnd.dataresource'
+	],
+	defaultRank: 40,
```

Comment:
Ask: check JupyterLab & NTeract's renderer priority and see if they've changed / how they prioritize these.

Reply:
I found the code for JupyterLab, but not for NTeract. Can you point me in the right direction there?

The numbers from JupyterLab are different, but it follows the same structure, where specific renderers have a rank that's lower than more generic renderers.